flambe/experiment/experiment.py

```diff
@@ -629,3 +634,27 @@ def parse(self) -> None:
                     "Experiment name should contain only alphanumeric characters " +
                     "(with optional - or _ in between)"
                 )
+
+    def get_user(self) -> str:
+        """Get the user that triggered this experiment.
+
+        Returns
+        -------
+        str:
+            The user as a string.
+
+        """
+        return self.env.local_user if self.env else getpass.getuser()
```

Comment:
Inject get.pass getuser() as a user_provider and you don't need to use patching in the test. You can leave it as default parameter so that there's no need to specify it in the yaml, but you also leave the posibility of external users setting their own users

Reply:
Agreed. Done.